Cargo.toml

```diff
@@ -0,0 +1,64 @@
+# Copyright (c) 2024 Contributors to the Eclipse Foundation
+#
+# See the NOTICE file(s) distributed with this work for additional
+# information regarding copyright ownership.
+#
+# This program and the accompanying materials are made available under the
+# terms of the Apache License Version 2.0 which is available at
+# https://www.apache.org/licenses/LICENSE-2.0
+#
+# SPDX-License-Identifier: Apache-2.0
+
+[workspace]
+resolver = "2"
+members = [
+    "up-streamer",
+    "examples/simple-example",
+]
+
+[workspace.package]
+rust-version = "1.66.1"
+version = "0.1.5-dev" # uProtocol version
+repository = "https://github.com/eclipse-uprotocol/uprotocol-platform-linux-zenoh"
```

Comment:
this URL seems wrong to me

Reply:
You're right! This is outdated from when this repo was called that :slightly_smiling_face: